recipes/esptool/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "esptool" %}
+{% set version = "2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1e4288d9f00e55ba36809cc79c493643c623bfa036d7b019a0ebe396284bc317
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - esptool.py = esptool:_main      # [win]
```

Comment:
Is there a reason why the entry points are used only for Windows? I believe they should be the same for all platforms?

Reply:
I‘ve replicated what’s in their setup.py: https://github.com/espressif/esptool/blob/master/setup.py#L64

Reply:
Thank you for the clarification!